### PR TITLE
FAQ: Replace html with markdown

### PR DESCRIPTION
**Pull request by davelab6**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### What does Noto mean?
 
-When text is rendered by a computer, sometimes characters display as little boxes because your device doesn't have a font that has images for them— these boxes are known as “tofu.”  The name Noto is to convey Google’s goal that users see “__no__ more <b>to</b>fu.”
+When text is rendered by a computer, sometimes characters display as little boxes because your device doesn't have a font that has images for them— these boxes are known as “tofu.”  The name Noto is to convey Google’s goal that users see “__no__ more __to__fu.”
```
Comment (dougfelt) [inline FAQ.md]: Github does not render this properly for me.  It was the first thing I tried.  Does it work for you?

Comment (davelab6) [inline FAQ.md]: \* works

Comment (dougfelt) [inline FAQ.md]: I get underscores, unless I leave a space before 'fu', in which case I get bold plus a space.

Comment (dougfelt) [inline FAQ.md]: to__fu

Comment (davelab6) [inline FAQ.md]: Hmm, my 2nd commit didn't update this comment. 

But oof, I rebased to squash my 2nd commit that used 2 \* and works:

screen shot 2016-03-25 at 22 35 44

But there are now a few recent commits by you and Sacha in this PR. Hmm.

**Comment by dougfelt** [comment]
Ok, askterisks work.  Should have tried that when underscores failed.  Closing since I did this in my own branch.

**Pull request closed without merging**